src/shipit_pulse_listener/shipit_pulse_listener/__init__.py

```diff
@@ -1,5 +1,6 @@
 from cli_common.log import init_logger
+import logbook
 
 
 # Setup logger for bot execution
-init_logger()
+init_logger(level=logbook.INFO)
```

Comment:
i'd just squeeze it into this one since we are anyway touching all of the ``init_logger`` invocations